snapshot/android/project-snapshot-generator.js

```diff
@@ -160,23 +165,33 @@ ProjectSnapshotGenerator.prototype.generate = function(generationOptions) {
         this.generateTnsJavaClassesFile({ output: tnsJavaClassesDestination, options: generationOptions.tnsJavaClassesOptions });
     }
 
+    var snapshotToolsPath = generationOptions.snapshotToolsPath ?
```

Comment:
**style**: let's make it an arrow func instead of using ternary operator 2 times